config/opensearch/startup.yml

```diff
@@ -0,0 +1,27 @@
+cluster.name: okpt-cluster
```

Comment:
leave this file as opensearch.yml. That's the standard name for the config file for opensearch

Reply:
Done. Filename is reverted